### DISCUSSION TITLE
Authentication Code Request Failure

### DISCUSSION THREAD

**Discussion by Arif-dev** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#697] [General]
I already follow the instruction step by step and find the trouble when i try to grant my email to trust this project. In case, instead of show me the code to copy inside the field, it show me an error like this.. Please help guyss, its almost done

![image](https://user-images.githubusercontent.com/54586635/163715466-5daad439-0943-4d30-819f-daf1e296dc76.png)

error unable to connect
![image](https://user-images.githubusercontent.com/54586635/163715329-18b625f8-62a0-4e3f-9fa9-c4c730b5d830.png)

**Comment by sumeshir26** [comment]
Copy the code from the address bar

**Reply by Arif-dev** [reply]
which one it is?
http://localhost/?code=4/0AX4XfWgEymbZHTSyyK6R-T356pCwmwVAMy94g5pcMJmDHRsodbzbWAGEG6XV0P7lMFJ3jg&scope=https://www.googleapis.com/auth/assistant-sdk-prototype

thx before...

**Comment by sumeshir26** [comment]
Tehtext after ?code=4/ and before &scope
